src/datachain/lib/dc.py

```diff
@@ -2418,6 +2419,7 @@ def export_files(
         signal="file",
         placement: FileExportPlacement = "fullpath",
         use_cache: bool = True,
+        link_type: Literal["copy", "symlink"] = "copy",
```

Comment:
Another thing I forgot to mention - if files fail to symlink, it will always fall back to `copy`.